src/main/java/org/reduct/client/ReductClient.java

```diff
@@ -17,15 +17,20 @@
  * Base class for all clients.
  */
 public abstract class ReductClient {
+   private static final String ERROR_400 = "Posted content bigger or smaller than content-length";
```

Comment:
The server sends text messages with the error descriptions. We don't need re-implement this.

Reply:
done